lib/find.coffee

```diff
@@ -72,6 +72,7 @@ module.exports =
         @projectFindPanel.hide()
       else
         @projectFindPanel.show()
+        @projectFindView.focusFindElement()
```

Comment:
Sure, that would make sense. I guess in that case we can do the same for other functions like ``show``. Most of these functions seem to share some actions:

* Create the view.
* Disable 'the other find panel'.
* Show/hide the correct find panel.
* Optionally focus a specific element.

I would propose the following changes:

1. Write a function that determines 'the other find panel' based on the panel to be activated.
2. Generalize functions so that they take a panel to activate and an optional action like 'focus input x' as arguments.

What do you think? I'll try to write some more code soonish.